reader.py

```diff
@@ -130,13 +140,20 @@ def get_csv_lines(filename=SOURCE_CSV_PATH, cols=COLNAMES):
 #           
 # 
 
+# COMMENT: I suppose here you are not using pandas because of memory concerns?
+# Because getting the rows as dictionaries is very easy in a pandas DataFrame

```

Comment:
I had an implementation, where I used pandas to import source data file, put it had several drawbacks of different severety:
- one source csv file has a line that is +column than the rest of the fail, it fails on pandas import 
- memory is concern
- typing is a bit opaque in pandas - I often get float as a result of operation on ints
- dtype is not compatible with skipfooter (C/python parsers)
-  SettingWithCopyWarning: A value is trying to be set on a copy of a slice from a DataFrame is annoying. 
... so as a sum of the above with a dictionary I feel more in control of what is happening. However dequote() and okved3() fucntions can easilly be done in pandas, I think. Each of this functions returns a tuple or a list, how can one use a column as a fucntion arguement and set other few columns as result of these functions?